### ISSUE TITLE
Renaming files within editor doesn't stick and reverts to "New File"

### ISSUE THREAD

**Issue by bradgy** [CollaboraOnline/richdocumentscode#55]
Potential issue/bug within the CODE server or upstream.

**Description**: After opening an existing file, renaming the file from within the editor doesn't save the new name, and instead changes the file name to New File.xxx where xxx is the existing file extension. File name can be changed back/renamed in Nextcloud after exiting the rich editor.

**Representative webm**: https://imgur.com/a/Sw3ZbCw

**Way to replicate**: Open existing file on Nextcloud server. Change filename within editor. Observe name change has not been effective.

**Expected behaviour**: When changing the name using the editor, the file name is immediately changed/saved reflecting the new value.

**System**: Nextcloud 19.0.1, Collabora Online 3.7.3, Docker containers: collabora/code:latest, linuxserver/nextcloud:latest, linuxserver/mariadb:latest. Everything behind traefik reverse proxy.

**Comment by timar** [comment]
@eszkadev you fixed this generally, but it seems that it's still reproducible with richdocumentscode. Maybe it's a special case because of the proxy? Could you please check it? Thanks!

**Comment by bradgy** [comment]
Hi timar - this is actually fixed for me; funnily enough I remembered this bug report about 2 weeks ago after a file name change saved successfully, but couldn't find the report via search for some reason.

I logged this report under richdocumentscode as I thought that referred to the CODE editors in Nextcloud. I was after a general fix, if this repo is a specific implementation separate to the main CODE project please ignore that it is here.

Thanks to everyone involved in fixing. Closing now.

**Comment by AnselmKoch** [comment]
> 

Hey bradgy, I currently have the same problem, do you mind telling me what the the cause for this behaviour was?

**Comment by bradgy** [comment]
No idea @AnselmKoch , but it appeared with my nextcloud collabora office traefik docker set up at the time, and an update to some part of the stack eventually fixed it. I have since used this feature successfully many times.

I just checked that this feature still works on my machine **_and it does not,_** so I may be encountering the same issue as you. Perhaps a recent update caused a regression.

Will reopen this and see if we can get some eyes on it.
